express-api/Dockerfile

```
@@ -0,0 +1,44 @@
+#############################################
+#                 Base                #
+#############################################
+# Use an official Node.js runtime as a base image
+FROM node:18.17.1-bullseye-slim as base
+
+# Set the working directory in the container
+WORKDIR /express-api
+
+ENV NODE_ENV=development
+ENV CONTAINERIZED=true
+
+# Copy files, excluding those in .dockerignore
+COPY . .
+
+# Install TypeScript. Needed for build process.
+RUN npm i -D typescript
+
+# Compile to JavaScript build 
+RUN npm run build
+
+#############################################
+#                 Prod Build                #
+#############################################
+FROM node:18.17.1-bullseye-slim as Prod
+
+# Set the working directory to /app
+WORKDIR /express-api
+
+ENV NODE_ENV=production
+ENV CONTAINERIZED=true
+
+# Install packages. Needed even for compiled build.
+COPY package.json .
+RUN npm i
+
+# Add curl for health check
+RUN apt-get update
+RUN apt-get install -y curl
+
+# Copy compiled build from base
+COPY --from=base /express-api/dist .
+
+CMD [ "node", "server.js" ]
```

Missing an end of file LF.
```suggestion
CMD [ "node", "server.js" ]

```

---

Included suggested change in the code.